### PR TITLE
Add donor facets

### PR DESCRIPTION
**Pull request by mccalluc**
Towards #671 , but does not provide any organization for the facet list.

### CODE REVIEW

File: context/app/static/js/components/Search/config.js [hunk 1]
```diff
@@ -27,7 +34,7 @@ export const sampleConfig = {
     filter('mapped_specimen_type', 'Specimen Type'),
     filter('donor.group_name', 'Group'),
     filter('created_by_user_displayname', 'Creator'),
-  ],
+  ].concat(makeDonorMetadataFilters(false)),
```
Comment (mccalluc) [inline context/app/static/js/components/Search/config.js:37]: @ngehlenborg : This and the one below add the four donor filters.

Comment (ngehlenborg) [reply context/app/static/js/components/Search/config.js:37]: Is this for sample only or also for datasets? We want it for both, I don't remember if I said that on Slack.

Comment (mccalluc) [reply context/app/static/js/components/Search/config.js:37]: Both: should have been more clear.

Comment (ngehlenborg) [reply context/app/static/js/components/Search/config.js:37]: Thanks!

**Pull request merged**